WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -70,6 +75,7 @@ private extension TotalsView {
     func subtotalFieldView(title: String, formattedPrice: String?, shimmeringActive: Bool, redacted: Bool) -> some View {
         if shimmeringActive {
             shimmeringLineView(width: Constants.shimmeringWidth, height: Constants.subtotalsShimmeringHeight)
+                .matchedGeometryEffect(id: "subtotalFieldView:_\(title)", in: totalsFieldAnimation)
```

Comment:
(This is unlikely to cause a problem, but might save us some future hassle if we can improve it easily now.)

`title` is a user-facing localized string – so we don't know exactly what it'll be at runtime.

It would be safer to use a defined string, perhaps the localized string key, as there's nothing to stop a translator using identical strings for multiple translations.

If the strings are duplicated, those fields are displayed on top of one another and become unreadable.

Also, it'd be good to define the prefixes in the `Constants` enum to avoid typos.

Reply:
True.

I'll pass matchedGeometryId through parameters, and use `UUID` for identifiers since they can have arbitrary values, most importantly they should be consistent and match between specific shimmering lines and text fields.

https://github.com/woocommerce/woocommerce-ios/pull/13437/commits/34ac5a2f908604b044e4bf8864f7466e878d29c0